docs/source/create/avatars/create-avatars.rst

```diff
@@ -33,14 +33,13 @@ Create an Avatar from Scratch
 
 The steps involved in creating your avatar are:
 
-1. Create an avatar with 3D character modeling tool such as Adobe Fuse, Blender or Maya. 
+1. Create an avatar with 3D character modeling tool such as Blender or Maya.
 2. Rig and animate your avatar with an animation tool such as Mixamo.
 3. Fine tune your avatar using a tool such as Blender or Maya.
-4. Package the model in Vircadia for use as an avatar.  
+4. Package the model in Vircadia for use as an avatar.
 
-Check out this YouTube playlist for one way to create and customize your own avatar. Here, we use Adobe Fuse to create our avatar, Mixamo to rig our avatar automatically, and Blender to adjust the rendering on our avatar. We also have written instructions on the same process: 
+Check out this YouTube playlist for one way to create and customize your own avatar. Here, we use Mixamo to rig our avatar automatically, and Blender to adjust the rendering on our avatar. We also have written instructions on the same process:
```

Comment:
I would say remove this line and the link to the YouTube video. The YouTube playlist will be extremely outdated, and if it depends on Adobe Fuse it will be of little use to most people.